fastmri/data/mri_data.py

```diff
@@ -358,3 +360,153 @@ def __getitem__(self, i: int):
             sample = self.transform(kspace, mask, target, attrs, fname.name, dataslice)
 
         return sample
+
+
+class AnnotatedSliceDataset(SliceDataset):
+    def __init__(
+            self,
+            root: Union[str, Path, os.PathLike],
+            challenge: str,
+            mri_type: str,
+            multiple_annotation_policy: str,
+            transform: Optional[Callable] = None,
+            use_dataset_cache: bool = False,
+            sample_rate: Optional[float] = None,
+            volume_sample_rate: Optional[float] = None,
+            dataset_cache_file: Union[str, Path, os.PathLike] = "dataset_cache.pkl",
+            num_cols: Optional[Tuple[int]] = None,
+            annotation_version: Optional[str] = None,
+    ):
+        """
```

Comment:
We need a description of the dataset here. Describe the fastmri-plus data set and provide a reference link to their paper.

Reply:
@Gaskell-1206, for the description of the dataset, create a new docstring right below the `AnnotatedSliceDataset` class definition, but before `__init__`.